initializers/sequelize.js

```diff
@@ -60,7 +68,8 @@ module.exports = {
         fs.readdirSync(dir).forEach(function(file){
           var nameParts = file.split("/");
           var name = nameParts[(nameParts.length - 1)].split(".")[0];
-          api.models[name] = api.sequelize.sequelize.import(dir + '/' + file);
+          var modelFunc = currySchemaFunc(require(dir + '/' + file));
+          api.models[name] = api.sequelize.sequelize.import(capitalizeFirstLetter(name), modelFunc);
```

Comment:
http://docs.sequelizejs.com/en/latest/docs/models-definition/#import

I'm following the design pattern happening in the second code block from this point in the docs. The first "name" argument is required to access the callback in the second arg but it doesn't appear to actually be used anywhere. I'll just strip that unneeded function and pass in the plain name instead.